PlagiarismPlugin.php

```diff
@@ -86,206 +91,208 @@ function getEnabled($contextId = null)
 			$ithenticate = new \bsobbe\ithenticate\Ithenticate($username, $password);
-		} catch (Exception $e) {
+		} catch (Throwable $e) {
 			$this->sendErrorMessage($submission->getId(), $e->getMessage());
 			return;
 		}
+
 		// Make sure there's a group list for this context, creating if necessary.
 		$groupList = $ithenticate->fetchGroupList();
 		$contextName = $context->getLocalizedName($context->getPrimaryLocale());
 		if (!($groupId = array_search($contextName, $groupList))) {
 			// No folder group found for the context; create one.
 			$groupId = $ithenticate->createGroup($contextName);
+
 			if (!$groupId) {
-				$this->sendErrorMessage($submission->getId(), 'Could not create folder group for context ' . $contextName . ' on iThenticate.');
+				$this->sendErrorMessage(
+					$submission->getId(), 
+					'Could not create folder group for context ' . $contextName . ' on iThenticate.'
+				);
+
 				return;
 			}
 		}
 
 		// Create a folder for this submission.
-		if (!($folderId = $ithenticate->createFolder(
+		$folderId = $ithenticate->createFolder(
 			'Submission_' . $submission->getId(),
 			'Submission_' . $submission->getId() . ': ' . $publication->getLocalizedTitle($publication->getData('locale')),
 			$groupId,
 			true,
 			true
-		))) {
-			$this->sendErrorMessage($submission->getId(), 'Could not create folder for submission ID ' . $submission->getId() . ' on iThenticate.');
+		);
+
+		if (!$folderId) {
+			$this->sendErrorMessage(
+				$submission->getId(), 
+				'Could not create folder for submission ID ' . $submission->getId() . ' on iThenticate.'
+			)
+			;
 			return;
 		}
 
-		$submissionFiles = Repo::submissionFile()
-			->getCollector()
-			->filterBySubmissionIds([$submission->getId()])
-			->getMany();
-
-		$authors = $publication->getData('authors')->toArray();
+		$submissionFiles = Services::get('submissionFile')->getMany([
+			'submissionIds' => [$submission->getId()],
+		]);
+		$authors = $publication->getData('authors');
 		$author = array_shift($authors);
+
 		foreach ($submissionFiles as $submissionFile) {
 			$file = Services::get('file')->get($submissionFile->getData('fileId'));
-			if (!$ithenticate->submitDocument(
+
+			$submittedDocumentIdentifier = $ithenticate->submitDocument(
 				$submissionFile->getLocalizedData('name'),
 				$author->getLocalizedGivenName(),
 				$author->getLocalizedFamilyName(),
 				$submissionFile->getLocalizedData('name'),
 				Services::get('file')->fs->read($file->path),
 				$folderId
-			)) {
-				$this->sendErrorMessage($submission->getId(), 'Could not submit "' . $submissionFile->getData('path') . '" to iThenticate.');
+			);
+
+			if (!$submittedDocumentIdentifier) {
+				$this->sendErrorMessage(
+					$submission->getId(), 
+					'Could not submit "' . $submissionFile->getData('path') . '" to iThenticate.'
+				);
 			}
 		}
+
+		return;
 	}
 
 	/**
 	 * @copydoc Plugin::getActions()
 	 */
-	function getActions($request, $verb)
+    function getActions($request, $verb)
```

Comment:
This is mixing space-based indentation and tab-based indentation; I'd suggest converting over to [PSR-12](https://www.php-fig.org/psr/psr-12/) when making changes to the style. Off the top of my head it should be possible to apply the PHP CS Fixer config file in OJS to plugin code as well.

Reply:
fixed.